python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
I'm not sure what you mean about the LocalFile being different to the DiracFile. The whole point of this is to make sure it's handled in the same way.
Adding more special checks for LocalFiles is just messy and puts more and more onus on a RTHandler developer who may not care about the mechanics of a IGangaFile but knows they want a file to be placed somewhere.
Altering the schema which is exposed to the user internally is something I'm keen to avoid as it causes more confusion than education imo if the user specifies something and then it changes because ganga 'knows better'